content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -9,176 +9,268 @@ With a down payment, the buyer pays a portion of the total amount owed while agr
    :align: center
    :alt: The deduct down payment option on the create invoices pop up in Odoo Sales.
 
-If the remaining amount is ready to be paid, the :guilabel:`Regular Invoice` option should be
-selected, and the :guilabel:`Deduct down payments` option should *stay* selected. Then, click
-:guilabel:`Create and View Invoice`.
+If the remaining amount is ready to be paid, select the :guilabel:`Regular Invoice` option. Odoo
+will create an invoice the exact amount needed to complete the total payment, as indicated in the
+:guilabel:`Amount to invoice` field.
+
+Then, click :guilabel:`Create Draft Invoice`.
 
-Doing so reveals a separate :guilabel:`Invoices` page, listing all the invoices for that specific
-sales order. Each invoice line item displays all the necessary information related to each invoice,
-along with the current :guilabel:`Payment Status`.
+Doing so reveals another :guilabel:`Customer Invoice Draft` page, listing *all* the invoices for
+that specific sales order in the :guilabel:`Invoice Lines` tab. Each invoice line item displays all
+the necessary information related to each invoice.
 
-.. image:: down_payment/down-payment-invoices-page.png
+.. image:: down_payment/second-down-payment-invoices-page.png
    :align: center
-   :alt: The down payment invoices page in Odoo Sales.
+   :alt: The second down payment invoices page in Odoo Sales.
 
-To complete the flow, select the invoice with the :guilabel:`Not Paid` status, which opens that
-invoice. On the invoice page, click :guilabel:`Confirm`, then :guilabel:`Register Payment`, and
-confirm that the remaining :guilabel:`Amount` is correct. Then, click :guilabel:`Create Payment`.
+To complete the flow, click :guilabel:`Confirm`, which changes the status of the invoice from
+:guilabel:`Draft` to :guilabel:`Posted`. Then, click :guilabel:`Register Payment`.
 
-That pop-up window disappears, and a green :guilabel:`In Payment` banner is now present on the
-invoice. The line items on the invoice show that the total amount has been paid, including mentions
-of the down payment, and the flow is complete.
+Once again, the :guilabel:`Register Payment` appears, with all fields auto-populated with the
+necessary information, including the remaining amount left to be paid on the order.
 
-This flow is also possible with the :guilabel:`Fixed amount` down payment option.
+.. image:: down_payment/second-register-payment-popup.png
+   :align: center
+   :alt: The second register payment pop-up form in Odoo sales.
+
+After confirming that information, click :guilabel:`Create Payment`. Doing so reveals the final
+:guilabel:`Customer Invoice` with a green :guilabel:`In Payment` banner in the upper-right corner.
+Also, both down payments are present in the :guilabel:`Invoice Lines` tab.
+
+.. image:: down_payment/second-down-payment-in-payment-invoice.png
+   :align: center
+   :alt: The second down payment invoice with in payment banner in Odoo Sales.
+
+And the flow is now complete.
```

Comment:
I think we could do without this sentence personally, but feel free to keep it if you think it helps to separate from the blocks below

Reply:
I reworded it slightly just to clearly indicate to the reader that the flow they just read through is now complete.